constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/helper/widget/Carousel.java

```diff
@@ -309,17 +360,19 @@ private void updateItems() {
         }
 
         final int count = mAdapter.count();
```

Comment:
can move the count variable inside the if block, or maybe simpler, add if (infiniteCarousel) { return; } before final int count

Reply:
Done ✅